ochre/CreateFigures.py

```diff
@@ -21,6 +21,7 @@
     'Gas Generator': 'purple',
     'Battery': 'c',
     'Other': 'grey',
+    'Lighting': 'orange',
```

Comment:
Not that it's a big issue, but how (if at all) should we try to capture smaller changes like this in PRs that are incidental? Hopefully over time there'll be a lot less of them. I have no objection to including these changes unmentioned because they're really minor and not worth separating into their own PR. Maybe what I have in the template about making sure the changelog is updated as appropriate is enough, and we'll just have to keep making case by case decisions about what is mentioned there?

Reply:
I'm not sure what is best practice, but I find that reviewing commit notes is a lot harder than PR notes when creating a changelog. So I think it's worth keeping most things in PRs, but tying them into larger PRs rather than creating one for each little change. 

Another option could be to update the changelog every time there's a change, rather than all at once when we create a new release. It still seems like some things will be so small that it's just not worth logging at all.

Reply:
Yeah I generally vote for option 2 (update the changelog as appropriate) with just the stuff big enough to be worth mentioning.